.gitignore

```diff
@@ -26,7 +26,7 @@ prime
 # The .vscode folder contains launch configuration and tasks you configure in
 # VS Code which you may wish to be included in version control, so this line
 # is commented out by default.
-#.vscode/
```

Comment:
because there are people use android studio, idx, vim, ...., so they don't like this :))

Reply:
So do you want me to comment this line ? 
I uncommented it since I use vscode and I saw the .idea file was in the .gitignore

Reply:
@hoangdat wdyt ? fine by me